src/main/java/com/google/sps/api/tasks/TasksClientHelper.java

```diff
@@ -48,11 +50,22 @@ public static String getMostRecentTaskListId(List<TaskList> taskList) {
   }
 
   /**
-   * Returns a task with the given due date.
+   * Returns an ExtendedTask with the given due date, with default duration.
    */
-  public static Task createTaskWithDue(DateTime dueDate) {
+  public static ExtendedTask createDefaultDurationTaskWithDue(DateTime dueDate) {
     Task task = new Task();
     task.setDue(dueDate.toStringRfc3339());
-    return task;
+    ExtendedTask extendedTask = new ExtendedTask(task, Scheduler.DEFAULT_DURATION_IN_MILLISECONDS);
+    return extendedTask;
+  }
+
+  /**
+   * Returns an ExtendedTask with specified due date and duration.
+   */
+  public static ExtendedTask createCustomDurationTaskWithDue(DateTime dueDate, Long duration) {
```

Comment:
I'm considering to refactor this method to just an inline task creation and call on ExtendedTask.setDuration()